applogic/src/api/types.rs

```diff
@@ -381,30 +380,15 @@ pub struct UiUserProfile {
     pub profile_picture_option: Option<Vec<u8>>,
 }
 
-impl From<UserProfile> for UiUserProfile {
-    fn from(user_profile: UserProfile) -> Self {
+impl UiUserProfile {
```

Comment:
Is there a reason you prefer this to `impl From<...>`?

Reply:
Two reasons:

1. `From` should be implemented for non-expensive conversions, which is not the case here. We are cloning.
2. `From` is always public and is hard to track, when it is unused.

Reply:
I checked the docs of `From` in the std lib again. Looks like we could leave the conversion via `From`. WDYT? I can restore it.